collector/src/main/java/com/bitmovin/analytics/BitmovinAnalytics.java

```diff
@@ -184,8 +186,14 @@ public void onError(ErrorCode errorCode) {
         Log.d(TAG, String.format("onError %s", playerStateMachine.getImpressionId()));
         EventData data = playerAdapter.createEventData();
         data.setState(playerStateMachine.getCurrentState().toString().toLowerCase());
+
+        if (playerStateMachine.getCurrentState() == PlayerState.SETUP) {
+            data.setVideoStartFailed(true);
+            data.setVideostartFailedReason(VideoStartFailedReason.PLAYER_ERROR.toString().toLowerCase());
+        }
```

Comment:
Do we really track this as a play attempt?
Does it count as a play attempt if the player errors during setup?

Reply:
If e.g. the first segment is corrupted then error will occur after user pressed play and player will never go into playing state. I'm not sure if there is some other case when error may happen after pressing play and actual start of playing. @peterpf what do you think does this case counts as play attempt?

Reply:
PlayAttempts should be only tracked after the `SETUP` finished. Errors before that are just normal errors, no playAttempt failures.